pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -102193,7 +102193,7 @@ self: {
        description = "Tool to help resolving git conflicts";
        license = stdenv.lib.licenses.gpl2;
        hydraPlatforms = stdenv.lib.platforms.none;
```

Comment:
@KaiHa @bhipple You also have to remove this `hydraPlatforms = ` line.  If you don't, then Hydra will not build this package.  You'll always have to build it locally.

Please send a follow-up PR removing this line.

Reply:
@cdepillabout @bhipple Done, you can find my PR [here](https://github.com/NixOS/nixpkgs/pull/102491).